eclass/kernel-2.eclass

```diff
@@ -290,17 +290,17 @@ handle_genpatches() {
 
 		local use_cond_start="" use_cond_end=""
 
-		if [[ "${i}" == "experimental" && -z ${K_EXP_GENPATCHES_PULL} && -z ${K_EXP_GENPATCHES_NOUSE} ]] ; then
+		if [[ ${i} == experimental && -z ${K_EXP_GENPATCHES_PULL} && -z ${K_EXP_GENPATCHES_NOUSE} ]]; then
 			use_cond_start="experimental? ( "
```

Comment:
Again, for consistency's sake I think we'll stick to `""` quoting ..

Reply:
Bash doesn't do any word splitting within `[[ ]]`. Quotes don't matter if they're there but it is redundant